data/pc/1.6/protocol.json

```diff
@@ -0,0 +1,2898 @@
+{
```

Comment:
TODO: Protocol has not been touched after copying from 1.7, so some things will inevitably be wrong.